src/AccessibilityInsights/Modes/LiveModeControl.xaml.cs

```diff
@@ -190,7 +190,7 @@ public async Task SetElement(Guid ecId)
                         this.ctrlHierarchy.DataContext = ec.DataContext;
                         this.ElementContext = ec;
 
-                        // make sure that  when selected node is changed in hiearchy tree, enable selector.
+                        // make sure that  when selected node is changed in hierarchy tree, enable selector.
```

Comment:
Remove extra space in this line?
```suggestion
                        // make sure that when selected node is changed in hierarchy tree, enable selector.
```